src/pages/Products/Products.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import { ProductCard } from "../../components";
+import "./Products.css";
+
+export const Products = () => {
+	const [products, setProducts] = useState([]);
+
+	useEffect(() => {
+		try {
+			(async () => {
+				const {
+					data: { products },
+				} = await axios.get("/api/products");
+				setProducts(products);
+			})();
+		} catch (error) {
+			console.log(error);
+		}
+	}, []);
+
+	return (
+		<div className="products-container flex-row">
+			{products.length ? (
+				<div className="products w-100 p-1 pt-5">
+					{products.map((product) => (
+						<ProductCard key={product.id} {...product} />
+					))}
+				</div>
+			) : (
+				<p className="not-available">No products available...</p>
+			)}
+		</div>
+	);
+};

```

Comment:
maybe mention page in the file name to prevent some confusion with the product card and it would make the file name more discriptive

Reply:
Since I only have Products.jsx inside pages and no other component, so for that I haven't named the file in that way